eth/tracers/native/call.go

```diff
@@ -188,7 +189,9 @@ func (t *callTracer) CaptureState(pc uint64, op vm.OpCode, gas, cost uint64, sco
 			return
 		}
 
-		log := callLog{Address: scope.Contract.Address(), Topics: topics, Data: hexutil.Bytes(data)}
+		position := len(t.callstack[len(t.callstack)-1].Calls)
```

Comment:
Isn't `position` same thing as `depth` ?

Reply:
I think they are not the same thing. here the log.position is used as relative to the number of calls in the current depth